spread_tests/integration/task.yaml

```diff
@@ -5,5 +5,6 @@ execute: |
   echo "Check that we are using the snapcraft snap"
   snapcraft_path="$(which snapcraft)"
   [ "$snapcraft_path" = "/snap/bin/snapcraft" ]
+  source /snapcraft/venv/bin/activate
```

Comment:
Yes, apparently it happens in a different environment or something. I added it just because without it, it failed.